src/lite-yt-embed.js

```diff
@@ -18,6 +18,9 @@ class LiteYTEmbed extends HTMLElement {
         // A label for the button takes priority over a [playlabel] attribute on the custom-element
         this.playLabel = (playBtnEl && playBtnEl.textContent.trim()) || this.getAttribute('playlabel') || 'Play';
 
+        this.title = this.getAttribute('title');
+        this.titleTag = this.getAttribute('title-tag') || 'div';
```

Comment:
just use a div. it's exactly what the real embed uses.

Reply:
Resolved by using your suggestion of stuffing the text into the ::before content value.